pysteps/blending/linear_blending.py

```diff
@@ -37,6 +38,7 @@ def forecast(
     end_blending=240,
     fill_nwp=True,
     nowcast_kwargs=None,
+    saliency=False,
```

Comment:
Perhaps we should put this line above the 'nowcast_kwargs=None' line - but that is more from a 'cleaner' code perspective.